client/src/utils/ApiUtils.jsx

```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
 
 // need to change  api server side to use /api
-const baseURL = 'http://localhost:3030/';
+const baseURL = 'http://server.bepwr.ca/';
```

Comment:
@meganniu 

you need to undo this locally post merge

Reply:
ok, thnx for the heads up